trade-finance-manager-api/api-tests/api.ts

```diff
@@ -106,4 +106,11 @@ export const createApi = (app: unknown): TestApi => ({
     }
     return requestInProgress.send(data);
   },
+  remove: async (url: string, data: AnyObject, { headers }: { headers: IncomingHttpHeaders | undefined } = { headers: undefined }) => {
```

Comment:
Would this work? To avoid defining `undefined`
```
headers?: IncomingHttpHeaders
```

Reply:
I think it makes sense to keep it consistent with the `get` and `put` methods above

Reply:
Makes sense, thanks